src/components/common/footer/Footer.jsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import './footer.scss';
+import './missionStatement.css'
```

Comment:
You don't need another .css file here. Take a look at the `footer.scss` file. You can just add `align-content: center` to the footer element already there. Then just add the `.missionStatement` class to the file as well.